CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to StackStorm pack directory
+
+You've decided to create or update a StackStorm integration pack. That's great! 
+We appreciate every contribution, and can't wait for your PR. To make sure other
+people (and robots) can use, extend and enjoy your work as well, we have some 
+guidelines in place, so please take a few minutes to read this doc through.
+
+You can get additional information about StackStorm packs and contributions
+in the StackStorm documentation:
+
+- [Integration Packs](https://docs.stackstorm.com/packs.html)
+- [Create and Contribute a Pack](https://docs.stackstorm.com/reference/packs.html)
+- [Pack testing](https://docs.stackstorm.com/development/pack_testing.html)
+
+## General guidelines
+
+* If you're contributing to a pack, please __update the version__ in `pack.yaml`! 
+  Without the version bump your PR will not be accepted. Pack versions follow 
+  [SemVer](http://semver.org/), and you should declare the new version 
+  accordingly.
+
+* If you're creating a brand-new pack, make sure all the __metadata__ is in place: 
+  the `pack.yaml` file, the necessary folder structure, ideally a 64x64 PNG icon.
+
+* It's important that all packs remain stable and reliable, so we recommend 
```

Comment:
I want to put it here since it's st2contrib specific and pack tests itself are not st2contrib specific - you could also write tests for your pack or whatever where those targets don't exist.